app/components/user-status.hbs

```diff
@@ -9,12 +9,10 @@
 <div class="buttons">
   {{#each this.currentUserStatus as |currentStatus|}}
     {{#if (eq @status currentStatus.status)}}
-      {{#if (not-eq @status 'ooo')}}
-        <button class={{currentStatus.class}} type="button" data-test-button disabled={{@isStatusUpdating}} {{on "click" (fn @onUpdateStatus currentStatus.firstAvailableStatus )}}>
+        <button class={{currentStatus.firstStatusClass}} type="button" data-test-button disabled={{@isStatusUpdating}} {{on "click" (fn @onChangeStatus currentStatus.firstAvailableStatus )}}>
           <span data-test-button-text>{{currentStatus.firstStatusMessage}}</span>
         </button>
-      {{/if}}
-      <button class="buttons--small" type="button" disabled={{@isStatusUpdating}} {{on "click" (fn @onUpdateStatus currentStatus.secondAvailableStatus )}}>
+      <button class={{currentStatus.secondStatusClass}} type="button" disabled={{@isStatusUpdating}} {{on "click" (fn @onChangeStatus currentStatus.secondAvailableStatus )}}>

```

Comment:
Can we please add a hover effect on these buttons, where on hover, the bg of the button changes to its accent color, and the text changes to white?
Add necessary CSS variables if needed for this behaviour.